server/swap/swap.go

```diff
@@ -753,11 +782,11 @@ func (s *Swapper) processInit(msg *msgjson.Message, params *msgjson.Init, stepIn
 	actor.status.swapVout = params.Vout
 	actor.status.swapTime = time.Now()
 	stepInfo.match.Status = stepInfo.nextStep
-	// NOTE: If UpdateMatch isn't non-blocking, this will need to be done
-	// differently.
-	s.storage.UpdateMatch(stepInfo.match.Match)
+	match := stepInfo.match.Match
 	s.matchMtx.Unlock()
 
+	s.saveMatch(match)
```

Comment:
I'm torn on whether we should be ignoring this error or not.

Reply:
I'll but a TODO on this so we can revisit it.